pg_search/tests/pg_regress/expected/test-fruit-types-issue.out

```diff
@@ -46,14 +46,15 @@ SELECT name, SUM(price), MAX(rating), AVG(age)
 FROM users 
 WHERE color @@@ 'blue' 
 GROUP BY name;
-                                                                                                   QUERY PLAN                                                                                                   
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+                                                                  QUERY PLAN                                                                   
+-----------------------------------------------------------------------------------------------------------------------------------------------
  Custom Scan (ParadeDB Aggregate Scan) on public.users
    Output: name, now(), now(), now()
    Index: idxusers
    Tantivy Query: {"with_index":{"query":{"parse_with_field":{"field":"color","query_string":"blue","lenient":null,"conjunction_mode":null}}}}
-   Aggregate Definition: {"group_0":{"terms":{"field":"name","size":65000,"segment_size":65000},"aggs":{"agg_0":{"sum":{"field":"price"}},"agg_1":{"max":{"field":"rating"}},"agg_2":{"avg":{"field":"age"}}}}}
```

Comment:
I honestly feel like dumping the whole aggregate definition here was useful for us, even if it was less useful for users.

Should we continue to do both? The "Applies to aggregates" line seems like it kindof just repeats the content of the query string, which isn't as helpful.